src/main/java/ee/buerokratt/ruuter/domain/steps/http/HttpQueryArgs.java

```diff
@@ -3,21 +3,18 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import java.util.HashMap;
 import java.util.Map;
 
 @Data
 @NoArgsConstructor
 public class HttpQueryArgs {
     private String url;
     private Map<String, Object> query;
-    private Map<String, String> headers;
+    private Map<String, Object> headers = new HashMap<>();
```

Comment:
This is probably a stupid question, but currently you made the headers all "Objects" over the code, is there a specific reason as to why? If they were strings, then you could lose a whole lot of "bolierplate code".

Reply:
This is for evaluating headers, evaluateScripts method input map value is Object.